src/group_theory/subgroup/basic.lean

```diff
@@ -1333,6 +1333,15 @@ end
 
 variable (H)
 
+/-- A subgroup is maximal if it is a proper subgroup that is not contained in
+any other proper subgroup -/
+def is_maximal := H < ⊤ ∧ (∀ H' : subgroup G, H < H' → H' = ⊤)
```

Comment:
This is the same as `is_coatom`.

Reply:
Ah, I as wondering why there was no definition so far. Will adjust this PR!